pkg/controller/mizar/mizar-arktos-network-controller.go

```diff
@@ -254,10 +254,15 @@ func createVpcAndSubnet(vpc, subnet string, dynamicClient dynamic.Interface) err
 // For the initial release of Arktos Mizar integration, user specified VPC CIDR is not supported
 // Generate VPC CIDR randomly for now.
 func generateVPCSpec(vpcName string) (int, *MizarVPC) {
+	// TODO: this is a quick solution to randomize VPC start ip address. Due to variously reasons, Arktos
+	//   needs randomize VPC start ip to prevent service ip collision for now.
+	// This is a simplified version to avoid reserved internal address - however, it may collision with real external ip address.
+	// Will log as an issue and solve in the future
 	// randomize ip start segment:
-	ipStart := ran.Intn(255) + 1 // IpStart range [1, 255]
-	// Simply not allow ip start from 10, 172, 192, 100 for well-known private range
-	if ipStart == 10 || ipStart == 172 || ipStart == 192 || ipStart == 100 {
+	ipStart := ran.Intn(100) + 11 // IpStart range [11, 99] - 20
```

Comment:
range is [11, 110], isn't it?

Reply:
Updated code and UT. Thanks for catch it.